maintenance_account/models/account_move.py

```diff
@@ -113,13 +114,20 @@ def _set_equipment_category(self):
             self.equipment_category_id = category.id
 
     def _prepare_equipment_vals(self):
+        equipment_name = self.name
+        description = False
+        if "\n" in self.name:
+            lf_index = self.name.index("\n")
+            equipment_name = self.name[:lf_index]
+            description = self.name[lf_index + 1 :]
         return {
             "move_line_id": self.id,
-            "name": self.product_id.name,
+            "name": equipment_name,
             "product_id": self.product_id.id,
             "category_id": self.equipment_category_id.id,
             "assign_date": self.move_id.date,
             "effective_date": self.move_id.date,
             "partner_id": self.move_id.partner_id.id,
             "partner_ref": self.move_id.ref,
+            "note": description,
```

Comment:
Shouldn't this be converted to HTML?

Reply:
It is not necessary as self.name is only text for this case.

Reply:
Yes, but `description` is HTML, so passing `Line 2\nLine 3` is incorrect. You should convert it to `Line 2<br/>Line 3` for example.

Reply:
done